### PR TITLE
Update mongo-express to version 0.40.0

### PR DESCRIPTION
**Pull request by knickers**
This update includes:
- Add CreateIndex Function
- Added bit of info in readme about dev
- Added cross-env
- Fixed collapsible JSON export and render
- Fixed use node path api to resolve modules
- Fixed escaping issue in nested objects
- Allow dash in DB name

### CODE REVIEW

**Comment by knickers** [comment]
```console
      ME_CONFIG_BASICAUTH_PASSWORD="" \
      VCAP_APP_HOST="0.0.0.0"
  
 -ENV MONGO_EXPRESS 0.38.0
 +ENV MONGO_EXPRESS 0.40.0
  
  RUN npm install mongo-express@$MONGO_EXPRESS
```

**Comment by knickers** [comment]
```console

$ bashbrew build mongo-express:0.40.0
Using bashbrew/cache:63c652b4e70278d78fa3117dfbc07b89ec63dc7d2ba1fb32acc408841d66f3fc (mongo-express:0.40.0)
Tagging mongo-express:0.40.0
Tagging mongo-express:0.40
Tagging mongo-express:latest

$ test/run.sh mongo-express:0.40.0
testing mongo-express:0.40.0
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

File: library/mongo-express [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # maintainer: Nick Cox <nickcox1008@gmail.com> (@knickers)
 
-0.38.0: git://github.com/mongo-express/mongo-express-docker@500a1db15deef043a6937302a1407b25369c8b4a
-0.38: git://github.com/mongo-express/mongo-express-docker@500a1db15deef043a6937302a1407b25369c8b4a
+0.40.0: git://github.com/mongo-express/mongo-express-docker@500a1db15deef043a6937302a1407b25369c8b4a
+0.40: git://github.com/mongo-express/mongo-express-docker@500a1db15deef043a6937302a1407b25369c8b4a
```
Comment (yosifkit) [inline library/mongo-express]: You forgot to bump the commit id. :wink:

Comment (knickers) [inline library/mongo-express]: Holy Shit:exclamation: I totally thought I did. Having problems with copy and past in my terminal. Sorry and thanks @yosifkit!

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
warning: insecure protocol git:// detected: git://github.com/mongo-express/mongo-express-docker
diff --git a/_bashbrew-list b/_bashbrew-list
index d6276fe..8dd93b9 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -1,3 +1,3 @@
-mongo-express:0.38
-mongo-express:0.38.0
+mongo-express:0.40
+mongo-express:0.40.0
 mongo-express:latest
diff --git a/mongo-express_latest/Dockerfile b/mongo-express_latest/Dockerfile
index 7496156..2545168 100644
--- a/mongo-express_latest/Dockerfile
+++ b/mongo-express_latest/Dockerfile
@@ -26,7 +26,7 @@ ENV ME_CONFIG_EDITORTHEME="default" \
     ME_CONFIG_BASICAUTH_PASSWORD="" \
     VCAP_APP_HOST="0.0.0.0"
 
-ENV MONGO_EXPRESS 0.38.0
+ENV MONGO_EXPRESS 0.40.0
 
 RUN npm install mongo-express@$MONGO_EXPRESS
 
```

</details>

**Comment by knickers** [comment]
Build test of #2995; 908da3af (`mongo-express`):

```console

$ bashbrew build mongo-express:0.40.0
warning: insecure protocol git:// detected: git://github.com/mongo-express/mongo-express-docker
Building bashbrew/cache:d000b8604ae930ae8a2c18324a6c77d5d07dfeeeabcf5078257f397c2229eaf4 (mongo-express:0.40.0)
Tagging mongo-express:0.40.0
Tagging mongo-express:0.40
Tagging mongo-express:latest

$ test/run.sh mongo-express:0.40.0
testing mongo-express:0.40.0
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Comment by yosifkit** [comment]
Just noticed that you are still using node 6.9; at some point you may want to update to a newer version of node, since that version is no longer maintained.  I also wonder if it might be beneficial to switch to (or add) an alpine based version. :innocent:

https://hub.docker.com/_/node/ for the most recent set of images.

**Pull request merged**